sonic-net/sonic-buildimage
# [Wistron] Add sw-to3200k and 6512-32r platform support

#### Why I did it
Add sw-to3200k and 6512-32r support for 202111
#### How I did it
Add code in device and platform folder for 6512-32r
Update sw-to3200k code both in device and platform folder
#### How to verify it
Install on Wistron device and run command to verify, please check attachment for log

Signed-off-by: RogerChang [Roger_Chang@wistron.com](mailto:Roger_Chang@wistron.com)
#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
[202111_sw_3200K_platform_enhance.log](https://github.com/Azure/sonic-buildimage/files/9098675/202111_sw_3200K_platform_enhance.log)
